src/javascript/app/pages/user/reset_password.js

```diff
@@ -19,31 +41,86 @@ const ResetPassword = (() => {
                 err_msg = localize('[_1] Please click the link below to restart the password recovery process.', error_code === 'InputValidationFailed' ? localize('There was some invalid character in an input field.') : response.error.message);
             }
 
-            $('#form_error_msg').text(err_msg);
+            $form_error_msg.text(err_msg);
             $form_error.setVisibility(1);
         } else {
-            $('#msg_reset_password').text(localize('Your password has been successfully reset. Please log into your account using your new password.')).setVisibility(1);
+            $msg_reset_password.text(localize('Your password has been successfully reset. Please log into your account using your new password.')).setVisibility(1);
             setTimeout(() => {
                 Login.redirectToLogin();
             }, 5000);
         }
     };
 
-    const onLoad = () => {
-        const form_id = '#frm_reset_password';
+    const unlinkResponseHandler = (response) => {
+        if (response.error) {
+            $reset_pw_binary_container.setVisibility(0);
+            const err_msg     = response.error.message;
 
-        FormManager.init(form_id, [
+            $msg_reset_password.setVisibility(0);
+            $form_error.find('a').setVisibility(0);
+            $form_error_msg.text(err_msg);
+            $form_error.setVisibility(1);
+        } else {
+            Dialog.alert({
+                id               : 'sent_email_success_dialog',
+                localized_message: localize('Your Deriv account is unlinked from [_1]. Use [_2]your email and password for future log in.', [social_signup_identifier, '<br />']),
```

Comment:
do we want to say deriv?

Reply:
Wrong copy, asked to design to update as well.